spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -51,40 +49,63 @@
       )
     end
 
-    it 'returns a ResultAdjudicator' do
-      proofing_result = proof
+    context 'remote proofing' do
```

Comment:
The reason this diff is so huge is that I restructured the test a bit. I placed all the initial tests that pertain to remote stuff into a new context block, and then placed all of the existing IPP related tests into an IPP centered context block.